builtins/amp-img.js

```diff
@@ -82,9 +95,7 @@ export class AmpImg extends BaseElement {
     if (this.img_) {
       return;
     }
-    if (!this.srcset_) {
```

Comment:
We can do minimal parsing. Do a regex search for `https?://[^ ]+`, preconnect to it.